### DISCUSSION TITLE
High CPU/Memory Utilization with Gmail gsuit account

### DISCUSSION THREAD

**Discussion by mukeshmishra996** [cefsharp/CefSharp#4609] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

117.2.20.0

### Operating System

Windows 11

### Architecture

x64

### .Net Version

4.8

### Implementation

WPF

### Reproduction Steps

1. Login into Gmail G-Suit Account(business account)
2. Redirect to Gmail inbox
3. Open system task manager.
4. Memory and CPU utilization is high(continuously memory utilization is increasing).
5. Open Dev Tool.
6. An API call/request is consistently failing and appears to be stuck in an endless loop of repeated attempts(not able to re-produce this issue in chrome browser).

![MicrosoftTeams-image](https://github.com/cefsharp/CefSharp/assets/14114368/54225628-4ae8-437b-b402-7a9b3d5ff34f)
<img width="959" alt="MicrosoftTeams-image (2)" src="https://github.com/cefsharp/CefSharp/assets/14114368/2c3380d6-9e44-4a55-939a-b86fba52c68a">

### Expected behavior

After the page has fully loaded, there should be reduction in memory and CPU usage.

### Actual behavior

After the page has fully loaded, there is a significant increase in memory and CPU utilization.

### Regression?

We have tested this with various versions of CefSharp, including the latest one.

### Known Workarounds

I am unable to find any work around to fix this issue. I'm having difficulty finding a solution to address this problem.

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
> ### Expected behavior
> After the page has fully loaded, there should be reduction in memory and CPU usage.

Google doesn't support using embedded browser to load it's services, they've been actively blocking them for some time. See https://github.com/cefsharp/CefSharp/discussions/3506#discussioncomment-611813

This is not something that we can support.

You'll need to debug the errors you are getting and see if there's some sort of workaround. 

CefSharp is just one of many CEF based browser. CEF has it's own support forum at https://www.magpcss.org/ceforum/index.php

**Reply by mukeshmishra996** [reply]
Can you please help to release the CPU/Memory utilization.
